client/src/partials/fiscal/fiscal.update.js

```diff
@@ -0,0 +1,79 @@
+angular.module('bhima.controllers')
+.controller('FiscalUpdateController', FiscalUpdateController);
+FiscalUpdateController.$inject = [
+    '$state', 'ScrollService', 'FiscalService', 'NotifyService', 'util'
+];
+
+/**
+ * This controller is responsible for creating a debtor group. It provides
```

Comment:
Please make sure these comments are clean.  These comments refer to the `debtor group` module.  This will block the PR, as specified in the [PR standards document](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub).